### ISSUE TITLE
Misc Accessibility problems

### ISSUE THREAD

**Issue by andrewgormley** [netbox-community/netbox#16852]
### NetBox version

v4.1.0-dev

### Feature type

Change to existing functionality

### Proposed functionality

- [x] (Racks) Rack elevation objects missing alt text [Multiple]
- [x] (Misc) Info hotspot missing link text [Multiple]
- [x] (Configure Table) some buttons are skipped when tabbing through the modal [Multiple]
- [ ] (Misc) When categorising objects by colour, we should have the colour name in the label or on hover [Multiple]
- [x] (Footer) Missing link text [Global]
- [x] (Dashboard) Module actions missing text [Single]

### Use case

Fixing web accessibility issues ensures NetBox is best placed to serve users using screen readers and other accessibility tools.

**Comment by andrewgormley** [comment]
Open PR #16977 consists of all changes apart from item 4: using the color name with the color role. @jeremystretch I will likely need help with this one, although the 'add role' form displays both the color and color name, once the option is selected the only information that is captured is the hexcode. therefore displaying the color choice name is a little tricky.
